### ISSUE TITLE
Adding to obsidian repository?

### ISSUE THREAD

**Issue by wesleylcai** [kraibse/obsidian-table-sorting#22]
Any plans on adding this to the plugins repository?

**Comment by kraibse** [comment]
Yes, there are plans to add it to the main plugin repository. Before resubmitting, I will have to do some general cleanup though.

**Comment by brayevalerien** [comment]
Is there a roadmap for the needed cleanup so we can contribute to it?

**Comment by kraibse** [comment]
You should be able to see the todo list in [this project kanban](https://github.com/users/kraibse/projects/2).
